### ISSUE TITLE
Handle invalid offset values in the meta file

### ISSUE THREAD

**Issue by brend32** [motion-canvas/motion-canvas#1036]
**Describe the bug**
The browser stucks when trying to load events

**To Reproduce**
Steps to reproduce the behavior
or a code snippet containing the scene that causes the problem:
1. Set targetTime to null in meta file
2. Open project

```ts
  EditableTimeEvents.ts line 100
      if (changed) {
        event = newEvent;
        this.lookup.set(name, event);
      }
    }

    this.registeredEvents.set(name, event);

   -> return event.offset; // NaN forces infinite loop without possibility to recover

       fix:
       return event.offset ?? 0;
  }
```

**Expected behavior**
Fallback to default values

**Console errors**
Browser not responding

**Package versions:**
3.15.2

- core: 3.15.2

**Comment by aarthificial** [comment]
Did the editor itself generate these `null` values? Or did you just inserted them manually to check what would happen?
If there's some weird behavior causing these `null`s to appear in the meta file we should also investigate that.

**Comment by brend32** [comment]
> Did the editor itself generate these null values?
I think no. I wanted to fix labels losing timings when using multiple scenes. And I changed signature from `targetTime` to `initialTime `and `duration`. When updating and moving projects from this version I forgot to delete meta file from second scene that caused infinite loop (two scenes had invalid meta but I deleted meta only for one scene).
Invistigated farther putting `null` don't break app, invalid number breaks. Also it will stuck for long time if you put large numbers, couse waitFor need yeald all this time to reach scene end. Don't know if there is easy fix, but if it is possible to give browser control back every 10 seconds it would be nice
